bin/node/src/config.rs

```diff
@@ -112,19 +112,22 @@ mod tests {
                         endpoint: Endpoint {
                             host: "127.0.0.1".to_string(),
                             port: 8080,
+                            protocol: Protocol::Http
```

Comment:
Could you update this test's endpoint inputs so that one of them remains the default (i.e. isn't specified), one is set to `"Http"`, and the other to `"Https"`. This should cover all cases.